### PR TITLE
Update README content and links

### PR DESCRIPTION
**Pull request by willingc**
Update the README file to clean up the markdown style, add more links to docs on ReadTheDocs, and reorder the sections:

* User Installation
* Running Jupyter notebook
* Development Installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,45 +4,84 @@
...
+language-agnostic **Jupyter notebook** in this repo and with the help of the
+community develop language specific kernels which are found in their own
+discrete repos.
+[![The Big Split™ announcement](https://blog.jupyter.org/2015/04/15/the-big-split/)]
+[![Jupyter Ascending blog post](http://blog.jupyter.org/2015/08/12/first-release-of-jupyter/)]
 
-## Usage
+## Installation
+Detailed [Installation documentation](http://jupyter.readthedocs.org/en/latest/install.html)
+can be found on [ReadTheDocs](http://jupyter-notebook.readthedocs.org/en/latest).
```
Comment (Carreau) [inline README.md]: I think it's confusing to have 1 sentence that link to 2 places. Not annoying for this PR in particular, but I would rephase in something like:

```
You can find the installation documentation for the [jupyter platform, on readthedocs](https://jupyter...)
The documentation for advance usage of Jupyter-Notebook can be found [here](https://jupyter-notebook.rtfd...)
```

**Comment by Carreau** [comment]
+1 as is.

Tip for screenshot, (doen't mean I don't like you theme with bird), I find that usually Safari allows to remove all window decoration (including address bar) which lead to much more compact screenshot vertically)

File: README.md (hunk 1 above)
Comment (willingc) [inline README.md]: @Carreau Good suggestion :sunny: Will add to pr

**Comment by willingc** [comment]
@Carreau The screenshot of bird theme can fly south for the winter. Will try to get a better shot on Safari and upload :smile:

**Comment by Carreau** [comment]
No, you can keep the screenshot.

**Comment by Carreau** [comment]
We should try an easier way to make screenshot anyway. Like chrome extension that resize the window to a specific determined width and screenshot the page. Then add the a false windows border when compiling with sphinx.

**Comment by willingc** [comment]
@Carreau Yeah, I was thinking the same thing about a predetermined width and also whether we wanted PNG or JPG for the screen shot. I'm happy to redo the screenshot. One advantage may be that a smaller shot would allow more viewing of the text below.

**Pull request merged**